### ISSUE TITLE
Crashing When Loading Chunk

### ISSUE THREAD

**Issue by Seikikai** [nekomaster1000/Infernal-Expansion#275]
**Bug:**
Game Crashing When Nether Chunks Load. When I Light The Nether Portal, It Starts Loading The Nether Chunks (Using [Immersive Nether Portals ](https://www.curseforge.com/minecraft/mc-mods/immersive-portals-for-forge)) The Game Crashes.

**Version Information:**
 - Operating System: Windows 10
 - Mod Version: 2.2.0
 - Minecraft Version: 1.16.5
 - Forge Version: 36.2.2
 
[crash-2021-09-02_14.37.19-server.txt](https://github.com/infernalexp/Infernal-Expansion/files/7101391/crash-2021-09-02_14.37.19-server.txt)

**Comment by SwanX1** [comment]
Are you able to repeat this issue consistently?

**Comment by Seikikai** [comment]
Yeah, Got 3 Crashes And All The Crash Logs Were Equal

**Comment by SwanX1** [comment]
Are you able to reproduce this issue with the latest version?

**Comment by Seikikai** [comment]
Sadly I Cannot Confirm If It Still Happening Or Not Because I Don't Have Anymore That World. I Been Creating With The Same Version That Caused The Crash Some Worlds But Due The Rarity Of That Biome Had To Create Instead Single Biome Worlds And Nothing Crashed.

**Comment by SelkeGore** [comment]
cannot confirm if it is still happening or not because he no longer has the same world. he's created worlds with the same version that initially caused the crash, but due to the rarity of the biome spawning he created a glowstone cavern only world and didn't have it reproduce.

tldr; he's not able to reproduce it
